src/app.js

```diff
@@ -95,6 +94,7 @@ class App extends Component {
     const input = {
```

Comment:
Unless these props are being used in multiple places, I think it's better to define them inline in the component like this:
```js
<LookupInput
  img="search"
  onChange={event => console.log(event.target.value)}
  onClick={() => console.log('clicked!')}
/>
```